validate.sh

```diff
@@ -2,16 +2,32 @@
 
 line='----------------------------------------'
 
-for sample in samples/*/*.json; do
-  printf "Validating %s %s " "$sample" "${line:${#sample}}"
-  result=$(python validate.py "$sample" rum-events-format.json 2>&1)
-  status=$?
-  if [ $status -ne 0 ]
-  then
-    echo "❌"
-    echo "$result"
-    exit $status
-  else
-    echo "✅"
-  fi
-done
+declare -a sample_schema_folders=(
+  "rum-schema"
+  "session-replay-schema"
+  )
+
+declare -a event_formats_for_samples=(
```

Comment:
Could not find a way to use `Map<String,String>` in bash so had to stick with arrays :)

Reply:
You could move this logic to `validate.py` and enjoy a more evolved language :)

Reply:
Yes, did not want to do this in this PR, maybe later in a dedicated PR.